quantum/action_tapping.c

```diff
@@ -334,8 +334,11 @@ bool process_tapping(keyrecord_t *keyp) {
                             .event.key     = tapping_key.event.key,
                             .event.time    = event.time,
                             .event.pressed = false,
+                            .event.event   = tapping_key.event.event,
+                            .event.combo   = tapping_key.event
+                                               .combo
 #    ifdef COMBO_ENABLE
-                            .keycode = tapping_key.keycode,
+                                               .keycode = tapping_key.keycode,
```

Comment:
Applying suggestions on deleted lines is not supported on GitHub so here's the diff of my suggestion instead.
```diff
diff --git a/quantum/action_tapping.c b/quantum/action_tapping.c
index 5c0be9c530..61025093d5 100644
--- a/quantum/action_tapping.c
+++ b/quantum/action_tapping.c
@@ -331,15 +331,15 @@ bool process_tapping(keyrecord_t *keyp) {
                         // unregister key
                         process_record(&(keyrecord_t){
                             .tap           = tapping_key.tap,
+#    ifdef COMBO_ENABLE
+                            .keycode = tapping_key.keycode,
+#    endif
                             .event.key     = tapping_key.event.key,
                             .event.time    = event.time,
                             .event.pressed = false,
                             .event.event   = tapping_key.event.event,
-                            .event.combo   = tapping_key.event
-                                               .combo
-#    ifdef COMBO_ENABLE
-                                               .keycode = tapping_key.keycode,
-#    endif
+                            .event.combo   = tapping_key.event.combo
                         });

```
On top of making it easier to read, it also solves the following compiler error occuring because of a missing comma after `tapping_key.event.combo`:
```
quantum/action_tapping.c: In function 'process_tapping':
quantum/action_tapping.c:341:48: error: request for member 'keycode' in something not a structure or union
  341 |                                                .keycode = tapping_key.keycode,
      |                                                ^
 [ERRORS]
```

Reply:
Thanks, fixed.